WISE-Community/WISE
# Issue 1229: Added static content to new Site homepage

How to test:
1. Load /site.
2. Make sure all the content on the new homepage loads correctly.

Let me know if you have any questions or suggestions! (I might update the photos eventually, once we have some new ones to use. May also add a few more minor animations to other content on the page.)
